luigi/rpc.py

```diff
@@ -116,31 +116,34 @@ def __init__(self, url='http://localhost:8082/', connect_timeout=None):
 
         self._rpc_retry_attempts = config.getint('core', 'rpc-retry-attempts', 3)
         self._rpc_retry_wait = config.getint('core', 'rpc-retry-wait', 30)
+        self._rpc_log_retries = config.getboolean('core', 'rpc-log-retries', True)
```

Comment:
Oh, this part of the code still doesn't use the [config class](http://luigi.readthedocs.io/en/stable/configuration.html#configuration-classes) syntax? Oh well maybe better done in a separate PR.